axelor-business-project/src/main/java/com/axelor/apps/businessproject/web/InvoicingProjectController.java

```diff
@@ -52,6 +52,7 @@
 	public void generateInvoice(ActionRequest request, ActionResponse response) throws AxelorException{
 		InvoicingProject invoicingProject = request.getContext().asType(InvoicingProject.class);
 		invoicingProject = invoicingProjectRepo.find(invoicingProject.getId());
+		invoicingProject.setStatusSelect(4);
```

Comment:
Please, never use hard coded value.

Reply:
we are already using hard coded values (mostly for select items) on most of xml views.